### ISSUE TITLE
Verification Email - Use Deep Linking to open app rather than website

### ISSUE THREAD

**Issue by jwpwhite** [PHPfox-Official/phpfox-v4-feature-requests#996]
Hi
For phpFox app users, the current registration process involves the following:

1. Register in app
2. Open email client and click on verification URL which opens a browser screen
3. Close browser, open app and login

The problem with the above process is that when a user clicks on the verification link, a web browser opens and they are directed to a login page. A whole bunch of users then login to the website rather than opening the app and logging in via the app. So it is not the best user experience as it can be confusing...

What I propose it to use deep linking so that the verification URL opens the app rather than a web browser. I feel that this will provide a much better user experience. E.g.

1. Register in app
2. Open email client and click on verification URL which opens the app
3. Login to app

Let me know your thoughts.

Kind regards
John

**Comment by sryn101** [comment]
Adding on to that, there should be deep linking for all apps as well, for example, when someone opens a group link and they have app installed already where they are logged in, they should be taken to mobile app instead of website.

**Comment by jwpwhite** [comment]
I agree @sryn101, deep linking across the board would be great!

**Comment by spirogg** [comment]
I agree as well.

**Comment by Myflag** [comment]
I agree as well.  This is an important issue for user verification and ease of registration.
